.github/workflows/java-publish.yml

```diff
@@ -40,6 +44,10 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Set up Node.js 16
```

Comment:
apparently this fixes the GLIBC version issue?

Reply:
This fixes the glibc issue in older ubuntu version, ubuntu version <= 20